python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -245,16 +213,8 @@ def aggregate(self, arg, split_every=None, split_out=1):
             else:
                 keys = self._meta.grouping.keys.name
 
-            return groupby_agg(
-                self.obj,
-                keys,
-                arg,
-                split_every=split_every,
-                split_out=split_out,
-                sep=self.sep,
-                sort=self.sort,
-                as_index=self.as_index,
-                **self.dropna,
+            return _make_groupby_agg_call(
+                self, keys, arg, split_every=split_every, split_out=split_out
```

Comment:
This is the one exception I see where `by` is not `self.by`. I wonder if it would make sense to always use `gb.by` in `_make_groupby_agg_call` and keep this one exception that calls `groupby_agg` directly.